src/sqlfluff/dialects/dialect_sparksql.py

```diff
@@ -805,8 +805,38 @@ class DatatypeSegment(PrimitiveTypeSegment):
     )
 
 
+# Object Reference Segments
+class CatalogReferenceSegment(ansi.ObjectReferenceSegment):
```

Comment:
can you add this content to dialect_databricks, and not spark?

Reply:
Thanks for the direction! I've moved all 'catalog'-related content to the Databricks dialect.